components/camel-casper/pom.xml

```diff
@@ -0,0 +1,383 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+   <parent>
+        <groupId>org.apache.camel</groupId>
+        <artifactId>components</artifactId>
+        <version>3.20.0-SNAPSHOT</version>
+    </parent>
+
+
+<artifactId>camel-casper</artifactId>
+    <packaging>jar</packaging>
+
+    <name>Camel :: Casper</name>
+    <description>Camel Casper blockchaine component</description>
+	<url>https://casperlabs.io/</url>
+
+
+	<properties>
+		<compiler.source>1.8</compiler.source>
+		<compiler.target>1.8</compiler.target>
+		<maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
+		<maven-resources-plugin.version>3.2.0</maven-resources-plugin.version>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+		<log4j2.version>2.13.3</log4j2.version>
+		<camel.version>3.14.2</camel.version>
+		<casper-java-sdk.version>0.3.0</casper-java-sdk.version>
+		<junit.version>4.13.2</junit.version>
+		<launchdarkly.version>2.3.2</launchdarkly.version>
+		<!-- SSE tests-->
+		<tomcat.version>9.0.54</tomcat.version>
+	</properties>
+
+	<licenses>
+		<license>
+			<name>MIT License</name>
+			<url>http://www.opensource.org/licenses/mit-license.php</url>
+		</license>
+	</licenses>
+
+	<developers>
+		<developer>
+			<id>mabahma</id>
+			<name>mabahma</name>
+			<email>elmabahma@gmail.com</email>
+		</developer>
+		
+	</developers>
+	<scm>
+		<connection>scm:git:git@github.com:abahmanem/camel-casper.git</connection>
+		<developerConnection>scm:git:git@github.com:abahmanem/camel-casper.git</developerConnection>
+		<url>https://github.com/abahmanem/camel-casper</url>
+	</scm>
+
+
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>org.apache.camel</groupId>
+				<artifactId>camel-bom</artifactId>
+				<version>${camel.version}</version>
+				<scope>import</scope>
+				<type>pom</type>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
+
+	<dependencies>
+
+		<!-- https://mvnrepository.com/artifact/org.apache.commons/commons-lang3 -->
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>
+			<version>3.11</version>
+		</dependency>
+		<!-- https://mvnrepository.com/artifact/commons-cli/commons-cli -->
+		<dependency>
+			<groupId>commons-cli</groupId>
+			<artifactId>commons-cli</artifactId>
+			<version>1.4</version>
```

Comment:
If possible / compatible should use `${commons-cli-version}` to be aligned with the rest of the code base.